python/doc/theory/meta_modeling/cross_validation.rst

```diff
@@ -634,12 +634,17 @@ The generic cross-validation method can be implemented using the following class
 - :class:`~openturns.KFoldSplitter`: uses the K-Fold method
   to split the data set.
 
-See :ref:`pce_cross_validation` for specific methods for the the cross-validation
-of a polynomial chaos expansion.
+Since the :class:`~openturns.LinearModelResult` is based on linear least
+squares, fast methods are implemented in the :class:`~openturns.experimental.LinearModelValidation`.
```

Comment:
```suggestion
Since :class:`~openturns.LinearModelResult` is based on linear least
squares, fast methods are implemented in :class:`~openturns.experimental.LinearModelValidation`.
```